### DISCUSSION TITLE
Feature Request : Migrate to Hardhat 2.10.0 Toolbox (Hardhat Chai Matchers, Hardhat Network Helpers)

### DISCUSSION THREAD

**Discussion by SvenMeyer** [PaulRBerg/hardhat-template#118] [Ideas]
Use Hardhat 2.10.0 Toolbox (Hardhat Chai Matchers, Hardhat Network Helpers)

https://github.com/NomicFoundation/hardhat/releases/tag/hardhat%402.10.0

**Comment by PaulRBerg** [comment]
Yup, completely onboard with this. Looks like @ahmedali8 implemented this in #119.
